### DISCUSSION TITLE
AlpineJS Sort Plugin is not working

### DISCUSSION THREAD

**Discussion by chrisvidal** [alpinejs/alpine#4393] [1. Help]
Hi,
I implemented the Sort plugin in my project and it is not working.
I can see its not working either here for instance https://codepen.io/wshayes/pen/PqWVwR

any idea what is missing guys ? thanks

**Comment by SimoTod** [comment]
No x-data

**Comment by chrisvidal** [comment]
I thought too but I have it in my code

```
<div x-data>
  <ul x-sort class="list-disc">
    {% for module in product.modules %}
      <li
        x-sort:item="{{ module.order_in_course }}"
        class="cursor-pointer">{{  module.name }}</li>
    {% endfor %}
  </ul>
</div>
```

it looks like AlpineJs is not running at all, I need to dig in the `app.js` code

**Comment by chrisvidal** [comment]
the js code cant be simpler

```js
import Alpine from "alpinejs";
import sort from "@alpinejs/sort";

Alpine.plugin(sort);

window.addEventListener("load", () => {});

```

**Reply by chrisvidal** [reply]
ok fixed missing `Alpine.start()`
